sys-apps/yarn/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+<email>fdsfgs@krutt.org</email>
+<name>soredake</name>
+</maintainer>
```

Comment:
You're missing proxy-maint: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#How_proxy_maintainership_works